stdlib/public/stubs/CMakeLists.txt

```diff
@@ -24,6 +22,7 @@ add_swift_library(swiftStdlibStubs OBJECT_LIBRARY TARGET_LIBRARY
   UnicodeExtendedGraphemeClusters.cpp.gyb
   ${swift_stubs_objc_sources}
   ${swift_stubs_unicode_normalization_sources}
+  TARGET_SDKS ${SWIFT_SDKS}
```

Comment:
Pretty sure that this hunk is bogus.  It is already a `TARGET_LIBRARY`, so I dont see why this needs `TARGET_SDKS` as well.  Thoughts @gribozavr?